modules/gdnative/include/gdnative/math_defs.h

```diff
@@ -36,10 +36,11 @@ extern "C" {
 #endif
 
 #include <stdint.h>
-
+#include <stdboo.h>
```

Comment:
Typo (should be `<stdbool.h>`).
`clang-format` will also complain about the order as it should be alphabetical, so it should be included on the line before `<stdint.h>`.

Reply:
Thanks, apologies